spec/forms/registration_number_forms_spec.rb

```diff
@@ -46,17 +50,74 @@
           registration_number_form.reg_identifier = ""
         end
 
+        it "is not valid" do
+          VCR.use_cassette("registration_number_form_valid_company_no", re_record_interval: 14.days) do
+            expect(registration_number_form).to_not be_valid
+          end
+        end
+      end
+    end
+
+    describe "#company_no" do
```

Comment:
After your opinion on this one. Should we / do we need also to have a test to cover someone entering a reg number as `1709418` rather than `01709418`?

Reply:
Yeah, I think this would be good to add.